README.rst

```diff
@@ -12,11 +12,10 @@ Zyte technologies (`scrapy-zyte-api
 Requirements
 ============
 
-* Python 3.8+
-* Scrapy 2.11+
-* zyte-spider-templates
```

Comment:
Removed since users are expected to install `requirements.txt` as part of the setup, they should not need to worry about this before.